### ISSUE TITLE
App blacks out when opening

### ISSUE THREAD

**Issue by kadry55** [letsdoitworld/World-Cleanup-Day#195]
South-Africa: 
1. Installed the app 
2. Tested that it was working at home without any problems
3. Once i tried to connect to the app outside, it just went to a blank screen - literally a black screen with the battery and time and connections at the top of the screen and nothing happened. 
4. Uninstalled the app at home and reinstalled it and i could go into it  fine once 
5. When I tried to go in again it went back to the blank screen.

phone details:
Samsung Galaxy Grand prime +
model number SM-G532F
Android version 6.0.1

**Comment by kadry55** [comment]
It appears that app works only after the first install. When it's closed down in the mean time it open with a black screen.

**Comment by KristiinaKerge** [comment]
new version fixes it, if some problems appears, please make new issue
